BT-Tracking/Interface/Active/ActiveAppVC.swift

```diff
@@ -139,6 +142,14 @@ final class ActiveAppVC: UIViewController {
         present(controller, animated: true)
     }
 
+    private func debugCancelRegistrationAction() {
+        AppDelegate.dependency.exposureService.deactivate { _ in
+            AppSettings.deleteAllData()
+            try? Auth.auth().signOut()
```

Comment:
Uz neni potreba

Reply:
To jsem tam dal kvuly testovani, kdyz pujdes s te predchozi verze aby se fakt opravdu vse smazalo

Reply:
To bychom meli resit plosne a ne jen pro testovaci pripady